test/EFCore.PG.FunctionalTests/Query/NorthwindFunctionsQueryNpgsqlTest.cs

```diff
@@ -282,6 +282,294 @@ public void Regex_IsMatch_with_unsupported_option()
             () =>
                 Fixture.CreateContext().Customers.Where(c => Regex.IsMatch(c.CompanyName, "^A", RegexOptions.RightToLeft)).ToList());
 
+    [Theory]
+    [MemberData(nameof(IsAsyncData))]
+    public async Task Regex_Replace_with_constant_pattern_and_replacement(bool async)
+    {
+        await AssertQuery(
+            async,
+            source => source.Set<Customer>().Select(x => Regex.Replace(x.CompanyName, "^A", "B")));
+
+        AssertSql(
+            """
+            SELECT regexp_replace(c."CompanyName", '^A', 'B', 'p')
```

Comment:
nit: this should be right-aligned like all the other SQL baselines (generally please try following existing style practices in an existing codebase).